stdlib/public/stubs/CommandLine.cpp

```diff
@@ -242,6 +242,39 @@ char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
 
   return argv;
 }
+#elif defined(__OpenBSD__)
+#include <sys/types.h>
+#include <sys/sysctl.h>
+#include <sys/exec.h>
+
+SWIFT_RUNTIME_STDLIB_API
+char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
+  if (_swift_stdlib_ProcessOverrideUnsafeArgv) {
+    *outArgLen = _swift_stdlib_ProcessOverrideUnsafeArgc;
+    return _swift_stdlib_ProcessOverrideUnsafeArgv;
+  }
+
+  int mib[2] = {CTL_VM, VM_PSSTRINGS};
+  struct _ps_strings _ps;
+  size_t len = sizeof(_ps);
+  if (sysctl(mib, 2, &_ps, &len, NULL, 0) != 0) {
+    return nullptr;
```

Comment:
It is never correct for this function to return `nullptr`. The WASI implementation here is wrong.

Reply:
Fixed.